backend/Jenkinsfile-develop

```diff
@@ -4,21 +4,15 @@ node('python') {
     stage('Unit Test') {
         checkout scm
         try {
-            sh 'cd backend && pip install --upgrade pip && pip install -r requirements.txt'
```

Comment:
@kuanfandevops can you confirm this works? I've noticed that `cd backend` was removed.

Reply:
You're right backend needs to be there!
I copied my code that does not do that.